prow/scripts/cluster-integration/update-kyma-gke-integration.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
```

Comment:
You may wanna refactor this script using the following helpers:
- cleanup-cluster.sh
- create-cluster.sh
- install-kyma.sh

Reply:
Hi @sayanh !
Unfortunately helper scrips assume existence of DNS and IP ranges on GCP and that will most likely cause errors.
I believe we can't use them without modifications.